README.md

```diff
@@ -1,4 +1,9 @@
 # statler
```

Comment:
It looks like there's a conflict in this file. Could you merge the new changes from dev into your branch and re-commit